core/src/main/java/io/kestra/core/schedulers/AbstractScheduler.java

```diff
@@ -891,7 +899,7 @@ private void logError(ConditionContext conditionContext, FlowWithSource flow, Ab
 
     private void sendWorkerTriggerToWorker(FlowWithWorkerTrigger flowWithTrigger) throws InternalException {
         FlowWithWorkerTrigger flowWithTriggerWithDefault = flowWithTrigger.from(
-            pluginDefaultService.injectDefaults(flowWithTrigger.getFlow(),
+            pluginDefaultService.injectAllDefaults(flowWithTrigger.getFlow(),
```

Comment:
Don't we already have the defaults as they are injected inside the flow listener consumer?